app/pages/HomePage/components/GuideList/GuideList.jsx

```diff
@@ -69,19 +69,30 @@ const GuideList = () => (
         </a>
       </li>
       <li className={styles.item}>
+        <a
+          href="/FoodMap"
+          // target="_blank"
```

Comment:
@JacobDFrank @richardxia can we also have the food pathway page open into a new tab in addition to the /covid page? Thank you!

Reply:
I commented that out because we haven't fixed the modal opening problem yet, it was discussed here. https://sheltertech.slack.com/archives/C0EGCTDCL/p1586646425205700

Reply:
Thanks @JacobDFrank! @AnnaKalkanis are you able to modify the GuideCard component to identify whether the link is a typeform or not? Want to have any non-typeform pages open a new tab on click from the homepage for google analytics tracking purposes. Thank you!

Reply:
Yup, I modified it and pushed it up on this PR. So, should be opening in a new tab now :)